vivisect/base.py

```diff
@@ -212,9 +216,23 @@ def _handleADDSEGMENT(self, einfo):
         self.segments.append(einfo)
 
     def _handleADDRELOC(self, einfo):
-        self.reloc_by_va[einfo[0]] = einfo[1]
+        rva, rtype = einfo
+        self.reloc_by_va[rva] = rtype
         self.relocations.append(einfo)
 
+        if rtype == RTYPE_BASERELOC:
+            fnm = self.getFileByVa(rva)
+            imgbase = self.getFileMeta(fnm, 'imagebase')
+            rbase = imgbase
+
+            ptr = self.readMemoryPtr(rva)
+            ptr += rbase
+            ptr &= e_bits.u_maxes[self.psize]
```

Comment:
If for some reason this did actually zero out some bits, that might be something that we would want to warn about.

Reply:
you're right, this is superfluous here.